assemblies/using-data-connections.adoc

```diff
@@ -0,0 +1,16 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="using-data-connections_{context}"]
+= Using data connections
```

Comment:
In this assembly, all of the modules titles are singular, and not plural, unlike the assembly above.